tasmota/my_user_config.h

```diff
@@ -782,8 +782,11 @@
 //#define USE_NOVA_SDS                             // Add support for SDS011 and SDS021 particle concentration sensor (+1k5 code)
   #define STARTING_OFFSET      30                // Turn on NovaSDS XX-seconds before tele_period is reached
 //#define USE_HPMA                                 // Add support for Honeywell HPMA115S0 particle concentration sensor (+1k4)
-//#define USE_SR04                                 // Add support for HC-SR04 ultrasonic devices (+1k code)
-  #define SR04_MAX_SENSOR_DISTANCE  500          // Set sensor max detection distance
+#define USE_SR04                                   // Add support for HC-SR04 ultrasonic devices (+1k code)
```

Comment:
As the current binaries are reaching max size pls disable both USE_SR04 (as was the case) and USE_ME007.

Reply:
Ok, done!